content/en/news/support/announcing-1.17/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Support for Istio 1.17 ends on October 27, 2023
+subtitle: Support Announcement
+description: Upcoming Istio 1.17 end of life announcement.
+publishdate: 2023-9-27
```

Comment:
I'm not seeing this on the deploy details. I'm wondering if the date needs to be 2023-09-27.

I'd also change the directory to content/en/news/support/announcing-1.17-eol

Reply:
It was the 0